cvat-ui/src/reducers/auth.reducer.ts

```diff
@@ -22,6 +22,55 @@ export default (
         isAuthenticated: false,
         error: action.payload,
       });
+
+    case 'LOGOUT':
+      return Object.assign({}, state, {
```

Comment:
Maybe it's better to use object destructuring instead of Object.assign, like eslint recommends. 
```{... object}```

Reply:
Where did you saw eslint recommendation?

Reply:
@zankevich 
Actually I've come across it [recently](https://github.com/opencv/cvat/pull/667/files#diff-e4403a877d80de653400d88d85e4801aR51). But I've disabled this rule for root directory because now we have legacy code there which isn't transpiled with babel. 
At the same time cvat-ui doesn't have it's own eslint settings, so it get settings from root config. I believe this rule is reasonable to use in modern ECMA-code.

Reply:
@zankevich 
It isn't requirements. It's more like an idea.